### PR TITLE
[ADD] attachment_unindex_content: New Module

### PR DESCRIPTION
**Pull request by ebirbe**
This module is intended to disable the indexation of data on the
ir.attachment model.

/cc @moylop260

### CODE REVIEW

**Comment by ebirbe** [comment]
@moylop260 I opened this PR here in `Vauxoo/server-tools` v12.0 to review it before open it in `OCA/server-tools`, but I'm realizing this repo in Vauxoo seems not to be completely prepared yet for 12.0 branch (causing red CI):

https://github.com/Vauxoo/server-tools/blob/12.0/.travis.yml#L19

Do you want me to prepare this repo for v12.0 or to open the PR directly in the OCA?

PD: Please, I'll appreciate any help to improve README.rst description if you consider.

/cc @keylor2906

**Comment by hugho-ad** [comment]
this should be tarjet to oca/v12.0

File: attachment_unindex_content/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from . import models
```
Comment (moylop260) [inline attachment_unindex_content/__init__.py:1]: Better using `hooks.py` like template:
 - https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py

File: attachment_unindex_content/README.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
+==========================
```
Comment (moylop260) [inline attachment_unindex_content/README.rst:1]: Better using readme folder like template:
 - https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Comment (moylop260) [reply attachment_unindex_content/README.rst:1]: I  just saw that it was created fine!
So Forget this comment

File: attachment_unindex_content/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# © 2019  Vauxoo (<http://www.vauxoo.com/>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Attachment Unindex Content',
+    'summary': 'Disable indexing of attachments',
+    'version': '12.0.0.0.0',
```
Comment (moylop260) [inline attachment_unindex_content/__manifest__.py:7]: Minimal `12.0.1.0.0`

File: attachment_unindex_content/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'author': 'Vauxoo',
```
Comment (moylop260) [inline attachment_unindex_content/__manifest__.py:8]: Vauxoo and OCA author

File: attachment_unindex_content/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    'category': 'Tools',
+    'depends': [
+        'base',
+    ],
+    'data': [],
+    'demo': [],
+    'installable': True,
+    'application': False,
+    'images': [],
+    'post_init_hook': '_clear_index_content',
```
Comment (moylop260) [inline attachment_unindex_content/__manifest__.py:20]: Better using `post_init_hook` method name like hooks.py from template:
 - https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py

**Comment by moylop260** [comment]
The module looks fine, just fix the small comments and create the new PR to OCA, Please

**Pull request closed without merging**